servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/DefaultHost.java

```diff
@@ -167,11 +167,11 @@ public boolean markExpired() {
             } else if (oldState.state == State.CLOSED) {
                 return true;
             }
-            Object nextState = oldState.connections.isEmpty() ? State.CLOSED : State.EXPIRED;
+            // Previous state was open.
```

Comment:
The other states are `ACTIVE` and `UNHEALTHY`. Do we need an assertion here instead of a comment?